packit_service/celerizer.py

```diff
@@ -47,6 +47,11 @@ def celery_app(self):
 
             # http://docs.celeryproject.org/en/latest/reference/celery.html#celery.Celery
             self._celery_app = Celery(backend=postgres_url, broker=redis_url)
+
+            days = int(getenv("CELERY_RESULTS_EXPIRATION", "30"))
```

Comment:
I'm wondering whether we should rather call it CELERY_RESULT_EXPIRES, which from docs is also the old name for result_expires.
I know I'm going into details, so I'll leave this on you.